common/static/common/js/discussion/utils.js

```diff
@@ -184,10 +184,8 @@
             if (!params.error) {
                 params.error = function() {
                     self.discussionAlert(
-                        gettext('Sorry'),
-                        gettext(
-                            'We had some trouble processing your request. Please ensure you have copied any ' +
-                            'unsaved work and then reload the page.')
+                        gettext('Error'),
```

Comment:
Yes. There are multiple instance of those `Error, Your request could not` I just deleted the Error, all across Discussion files.